### ISSUE TITLE
Migrate (debug) testing channel to Erythro Dev server

### ISSUE THREAD

**Issue by Ernest314** [ErythroGuild/polybius#18]
Might as well have a channel for testing in a centralized location (instead of how it was previously set up with multiple test servers and a testing channel in a different server).

**Comment by Ernest314** [comment]
This is no longer needed once #102 is finished.
